lib/src/main/java/com/futurewei/alcor/common/entity/DeviceOwner.java

```diff
@@ -0,0 +1,22 @@
+package com.futurewei.alcor.common.entity;
```

Comment:
It is a good idea to put all common entity classes out of an invidiual microservice. 

Instead of storing them in AlcorCommonLib, we recommend to store the entity classes in AlcorWeb which is under /web directory.